pyquil/tests/conftest.py

```diff
@@ -216,6 +216,21 @@ def _str_to_bool(s):
 def pytest_addoption(parser):
     parser.addoption("--use-seed", action="store", type=_str_to_bool, default=True,
                      help="run operator estimation tests faster by using a fixed random seed")
+    parser.addoption("--include-slow-tests", action="store_true", default=False, help="run tests marked as being 'slow'")
+
+
+def pytest_configure(config):
+    config.addinivalue_line("markers", "slow: mark test as slow to run")
+
+
+def pytest_collection_modifyitems(config, items):
+    if config.getoption("--include-slow-tests"):
+        # --runslow given in cli: do not skip slow tests
```

Comment:
what does this mean

Reply:
I think i see https://docs.pytest.org/en/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option

Reply:
if we do this, let's just use the canonical `--runslow`?